subsys/bluetooth/mesh/sensor_srv.c

```diff
@@ -473,6 +477,20 @@ static int cadence_set(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
 	sensor->state.min_int = min_int;
 	sensor->state.pub_div = period_div;
 	sensor->state.threshold = threshold;
+	sensor->state.configured = true;
```

Comment:
Should this be unset if the cadence is set back to its initial state?

Reply:
On reset only I guess. The spec doesn't define way to unset the cadence state via any message.

Reply:
Sensor Cadence settings can be made effect-less by setting Fast Cadence Period Divisor to zero.